Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/ComponentEditorHeader.hxx

```diff
@@ -45,6 +45,13 @@ namespace AzToolsFramework
         /// Set an icon. Passing a null icon will hide the widget.
         void SetIcon(const QIcon& icon);
 
+        /// Set a secondary icon to be drawn on top of the main icon.
+        void SetIconOverlay(const QIcon& icon);
+
+        // Set whether the component icon is clickable
+        void SetComponentIconClickable(bool clickable);
+        bool IsComponentIconClickable() const;
```

Comment:
Looks like this is not being used in the PR. Can be deleted?

Reply:
SetComponentIconClickable is being used in EntityPropertyEditor.cpp. Are you suggesting to remove the getter method?

Reply:
I meant the 'IsComponentIconClickable()' method

Reply:
OK, I typically like to have an accompanying getter method but I can remove it.